ci/submit-coverage

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Script to submit simplecov coverage report to Code Climate.
+# Only one report can be submitted per commit, so this can only
+# run once pre build, not for each test set.
+
+# The rspec and cucumber reports are all written to repo_root/coverage
+# Simplecov handles locking and merging for parallel tests.
+# The GKE tests can't mount that dir, so they produce a separate report
+# which is merged into repo_root/coverage by this script.
+
+set -eux
+
+dir="coverage"
+bin="cc-test-reporter"
+report="${dir}/.resultset.json"
+
+if [[ ! -e ${report} ]]; then
+    echo "SimpleCov report (${report}) not found"
+    ls -laR ${dir}
+    return 1
```

Comment:
Hmm, is there a reason why we don't use `exit <code>` instead of return?

Reply:
Nope, that should be `exit`, thanks.

Reply:
Still needs fixing